src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Properties/VsLangProjectPropertiesProvider.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.ComponentModel.Composition;
+using VSLangProj;
+
+namespace Microsoft.VisualStudio.ProjectSystem.VS.Properties
+{
+    /// <summary>
+    /// This provider imports <see cref="VSLangProj.VSProject"/> provided by CPS
+    /// and wraps it into an object that implements both <see cref="VSLangProj.VSProject"/> and 
+    /// <see cref="VSLangProj.ProjectProperties"/>. This enables us to provide
+    /// ProjectProperties to the Project Property Pages and maintain Backward Compatibility.
+    /// </summary>
+    internal class VsLangProjectPropertiesProvider
```

Comment:
Why does this class even exist, why can't we just export the VsLangProjectProperties?